### PR TITLE
fix loadMemDoc issue with records being added to multiple collections

### PR DESCRIPTION
**Pull request by mtCarto**
[NRPT-496](https://bcmines.atlassian.net/browse/NRPT-496)

### CODE REVIEW

File: api/migrations/20200709183253-loadMemDocs.js [hunk 1]
```diff
@@ -215,38 +222,40 @@ exports.up = async function (db) {
...
                 console.log(`Found record ${existingDoc._id} with a bad collection id ${existingDoc.collectionId}, adding to proper collection: ${bcmiCollection._id}`)
 
                 existingDoc.collectionId = bcmiCollection._id;
                 await nrpti.findOneAndUpdate({ _id: existingDoc._id }, existingDoc);
                 // duplicate record prevention
-                const arrayIncludes = bcmiCollection.records.some(item => item.toString() === existingDoc._id.toString());
+                const arrayIncludes = bcmiCollection.records.some(item => new ObjectID(item) === new ObjectID(existingDoc._id));
                 if (!arrayIncludes) {
-                  console.log(`Adding doc ${existingDoc._id} to docsArray of ${bcmiCollection._id}`);
+                  // console.log(`Adding doc ${existingDoc._id} to docsArray of ${bcmiCollection._id}`);
```
Comment (KitArmstrong) [inline api/migrations/20200709183253-loadMemDocs.js:234]: Still needed?

File: api/migrations/20200709183253-loadMemDocs.js [hunk 2]
```diff
@@ -302,10 +314,10 @@ async function createMineDocument(nrpti, nrptiMine, collection, collectionDoc, n
   // fetch doc from mem-admin Minio
   // minioObject contains de-serialized data returned from the getObject request.  Body field contains the data buffer
   // https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#getObject-property
-  const minioObject = await minio.getObject({
-    Bucket: MINIO_BUCKET,
-    Key: collectionDoc.document.internalURL
-  }).promise();
+  // const minioObject = await minio.getObject({
```
Comment (KitArmstrong) [inline api/migrations/20200709183253-loadMemDocs.js:317]: Still commented out.

**Review by KitArmstrong — APPROVED** [review]
I didn't see it occurring after two local runs.

**Review by MaxWardle — APPROVED** [review]
Tested that collections and documents do not duplicate in the database after multiple runs. I like the added logging, and all the checks. Nice job!

**Pull request merged**